pandas/tests/series/methods/test_inplace_ops.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from pandas import Series
+import pandas._testing as tm
+
+
+@pytest.mark.parametrize(
+    "dtype1, dtype2, dtype_expected, dtype_mul",
+    (
+        ["Int64"] * 4,
+        ["float"] * 4,
+        ["Int64"] + ["float"] * 3,
+        pytest.param(
+            "Int64",
+            "Float64",
+            "Float64",
+            "Float64",
+            marks=pytest.mark.xfail(reason="Not implemented yet"),
+        ),
+    ),
+)
+def test_series_inplace_ops(dtype1, dtype2, dtype_expected, dtype_mul):
```

Comment:
lets put this in tests.series.test_arithmetic (if there isnt already a class for inplace methods, lets make one)

Reply:
Done one (created the class + moved this)

Reply:
looks like test was copied, not moved.

Reply:
right - deleted now